E01-Geometry/Benas/src/shapes/Circle.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+
+class Circle extends Shape
+{
+    public float $radius;
+    public float $pi = 3.14;
+
+    public function __construct(float $radius)
+    {
+        if($radius>0) {
```

Comment:
Invert this `if` and make your code a bit more readable. Fail early rule.

```suggestion
        if($radius <= 0) {
            throw new InvalidArgumentException("Radius cannot be less than 0");
        }
        $this->radius = $radius;
```